docs/templates/tags.md

```diff
@@ -0,0 +1,91 @@
+# Tags
+
+The feature store enables users to attach tags to artifacts such as feature groups or training datasets. Tags are aditional metadata attached to your artifacts and thus they can be used for an enhance full text search (by default Hopsworks makes all metadata searchable, users can opt out for particular featurestores if they want to keep them private). Adding tags to an artifact provides users with a more dynamic metadata content that can be used for both storage as well as enhancing artifact discoverability.
```

Comment:
`feature stores`. Also the stuff between parenthesis breaks the flow of explaining what tags are and why they are useful.